.gitignore

```diff
@@ -159,9 +159,15 @@ cython_debug/
 #  option (not recommended) you can uncomment the following to ignore the entire idea folder.
 .idea/
 
+# Visual Studio Code
```

Comment:
similar to below, add a comment explaining what files vscode generates that we don't want in git.